src/utils/Types.tsx

```diff
@@ -14,15 +12,15 @@ export type Dimensions = {
   scale: number;
 };
 export type StateProp =
-  | row[]
-  | combinedRow[]
+  | Row[]
+  | CombinedRow[]
   | string
-  | combinedRow
```

Comment:
Got rid of combinedRow because currentCombinedRow is no longer a piece of state (replaced by currentCombinedRows)